README.md

```diff
@@ -1003,6 +1003,48 @@ const MyResponsiveDiv = glamorous.div({
 ```
 </details>
 
+In addition to all this, you can reference a `glamorous` component inside the
+styles of other `glamorous` components (this works because they each implement
+the `toString` method):
+
+<details>
+<summary>Contextual Overrides</summary>
+
+This API is inspired by `styled-components`.
+
+```javascript
+const Link = glamorous.a({
```

Comment:
Ah, gotcha. I'm on the fence here. Mostly because you don't have to do anything extra with styled-components and I don't want it to be any harder to do this with glamorous than it is with styled-components (which is why I added the plugin). If we wanted it to be harder, then I could just remove the plugin and you'd have to do: `glamorous.div({ [`& ${RedDiv}`]: {/*styles*/}})` which would be slightly annoying 🤔